src/notation/view/notationpaintview.cpp

```diff
@@ -652,6 +652,7 @@ void NotationPaintView::mousePressEvent(QMouseEvent* event)
 {
     setFocus(true);
     emit activeFocusRequested();
+    forceActiveFocus();
```

Comment:
Necessary or NotationPaintView won't receive ShortcutOverride events at all. Arguably it should forceActiveFocus the moment it's first displayed too?